src/components/forms/costs/__test__/CostsForm.test.tsx

```diff
@@ -62,7 +62,8 @@ describe('CostsForm Test', () => {
     const inputs = screen.queryAllByRole('textbox');
     expect(inputs.length).toEqual(0);
 
-    const [buttonSelect, buttonCancel, buttonSave] = screen.getAllByRole('button');
+    const [buttonCancel, buttonSave] = screen.getAllByTestId(/btn-/);
+    const buttonSelect = screen.getByRole('combobox');
```

Comment:
I've added a comment on how the process works.
In order to get the buttons i had to put some testid and get them with prefix `btn-`. 

The `select` role is 'combobox' because it refers to the mui's component as you can see in the [https://mui.com/material-ui/react-autocomplete/](documentation).